saleor/plugins/webhook/tests/test_shipping_webhook.py

```diff
@@ -422,8 +425,9 @@ def test_parse_excluded_shipping_methods_response(app):
 
     # then
     assert len(excluded_methods) == 2
-    assert excluded_methods[0]["id"] == "2"
-    assert excluded_methods[1]["id"] == external_id
+    assert excluded_methods[0].id == "2"
+    assert excluded_methods[1].id == external_id
+    assert mocked_schema_logger.call_count == 3
```

Comment:
we really want to log these? 🤔

Reply:
There are logs for invalid entries, and it was logged before, I think that we can provide such logs to customers in case the response data are wrong, and we are skipping them

Reply:
If it was logged before, we can leave it to avoid unrelated scope. But I think this would be eventually removed and we should use observability mechanism for that (like metric `saleor.webhook.shipping.errors`)